capstone/src/main/java/com/google/univiz/scorecard/URLProviderImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.google.univiz.scorecard;
+
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.Collectors.toCollection;
+
+import com.google.gson.annotations.SerializedName;
+import com.google.univiz.api.CollegeId;
+import com.google.univiz.config.UnivizConfig;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import javax.inject.Inject;
+
+final class URLProviderImpl implements URLProvider {
+  private final UnivizConfig univizConfig;
+  private final Set<String> fields;
+
+  @Inject
+  protected URLProviderImpl(UnivizConfig univizConfig) {
```

Comment:
I don't think you need to make it protected. Protected means by sub-classes and package classes. Since this is a final class, it cannot have sub-classes. So package-private is all you need.
```suggestion
  URLProviderImpl(UnivizConfig univizConfig) {
```